src/ros_filter.cpp

```diff
@@ -883,6 +885,11 @@ namespace RobotLocalization
       }
     }
 
+    // Check if the filter should start or not
+    nhLocal_.param("disabled_at_startup", disabledAtStartup_, false);
+    if (!disabledAtStartup_) enabled_ = true;
```

Comment:
Nitpick: would you mind making sure that (a) we don't nest braces (like in lines 1780, 1981, etc.) and (b) we don't do in-line if statements? As much as this package violates ROS coding standards, I do try in general to follow the same standard within the package.